awkward-cpp/CMakeLists.txt

```diff
@@ -105,6 +102,7 @@ target_link_libraries(awkward PRIVATE awkward-cpu-kernels-static ${CMAKE_DL_LIBS
 target_link_libraries(awkward PUBLIC awkward-parent)
 
 # Third tier: Python modules.
+find_package(Python REQUIRED COMPONENTS Interpreter Development.Module)
```

Comment:
Doesn't `pybind11` handle this for us? We don't link against these targets in this module.

Reply:
It defaults to the old FindPythonLibs/FindPythonInterp. You can use a variable to force the new search, or just search for it first.

If you build outside of scikit-build-core and want PyPy support, it's better to use the variable.

Reply:
(this is going to matter in CMake 3.27 which removes FindPythonLibs/FindPythonInterp if you set 3.27 as your max or min version)